benchmarks/bm/iast_fixtures/str_methods.py

```diff
@@ -11,20 +11,14 @@
 import random
 import re
 import threading
-from typing import TYPE_CHECKING  # noqa:F401
-from typing import Any  # noqa:F401
-from typing import Optional  # noqa:F401
-from typing import Text  # noqa:F401
-
-
-if TYPE_CHECKING:  # pragma: no cover
-    from typing import Callable  # noqa:F401
-    from typing import Dict  # noqa:F401
-    from typing import Generator  # noqa:F401
-    from typing import Iterable  # noqa:F401
-    from typing import List  # noqa:F401
-    from typing import Sequence  # noqa:F401
-    from typing import Tuple  # noqa:F401
+from typing import Callable
```

Comment:
why remove the if TYPE_CHECKING?

Reply:
cause we don't need it anymore, it's related to type hinting comments